lib/provider/ui/index.tsx

```diff
@@ -43,17 +43,17 @@ const Main = styled.main`
 `;
 
 export function UI({ children }: PropsWithChildren): ReactNode {
+  const [theme] = useAtom(themeAtom);
```

Comment:
Please use `useAtomValue` instead of `useAtom` because we don't need setter in this case.